.travis.yml

```diff
@@ -1,30 +1,63 @@
 language: rust
+cache: cargo
 rust:
-- nightly
+- stable
 sudo: required
 os:
 - linux
-rust:
-- nightly
-before_script:
-- rustup component add clippy
-- rustup component add rustfmt
+
+matrix:
+  include:
+    - name: "ARM Build"
+      env: 
+      - TARGET=arm-unknown-linux-gnueabihf
+      - RUSTFLAGS="-C linker=arm-linux-gnueabihf-gcc -L/usr/arm-linux-gnueabihf/lib -L$TRAVIS_BUILD_DIR/deps_root/usr/lib/arm-linux-gnueabihf"
+      - PKG_CONFIG_ALLOW_CROSS=1
+      - OPENSSL_DIR=$TRAVIS_BUILD_DIR/deps_root/usr
+      - OPENSSL_LIB_DIR=$TRAVIS_BUILD_DIR/deps_root/usr/lib/arm-linux-gnueabihf
+      - OPENSSL_INCLUDE_DIR=$TRAVIS_BUILD_DIR/deps_root/usr/include/arm-linux-gnueabihf
+      - SHORT_TARGET=armv6
+      addons:
+        apt:
+          packages:
+          - gcc-arm-linux-gnueabihf
+          - libc6-armhf-cross
+          - libc6-dev-armhf-cross
+          - libasound2-dev
+          - libssl-dev
+          - libssl1.0.0
+      before_script:
+      - echo "deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports/ trusty main" | sudo tee -a /etc/apt/sources.list
+      - sudo apt-get update
+      - sudo apt-get download libasound2:armhf
```

Comment:
Is there a reason why you don't just install the packages directly instead of unpacking them into a directory manually? I am currently trying to add more builds and it's actually a huge pain in the ass because I get huge a huge amount of errors. I actually can't install `libc6:armhf` because it doesn't match all versions of packages exactly. Even a patch mismatch fails the build...